apps/ledger-live-desktop/src/renderer/families/internet_computer/ManageNeuronFlowModal/steps/ListNeuron.tsx

```diff
@@ -0,0 +1,168 @@
+      );
+      setNeedsRefresh(false);
+      transitionTo("device");
+    }
+  }, [needsRefresh, transitionTo, account, onChangeTransaction, setNeedsRefresh]);
+
+  const onClickManage = useCallback(
+    (index: number) => {
+      if (account.type !== "Account") return;
+      setManageNeuronIndex(index);
+      transitionTo("manage");
+    },
+    [account.type, transitionTo, setManageNeuronIndex],
+  );
+
+  if (neurons) {
+    return (
+      <Container>
+        <TrackPage
+          category="Manage Neurons ICP Flow"
+          name="Step Confirmed"
+          flow="stake"
+          action="listNeurons"
+          currency={currencyId}
+        />
+        <SyncOneAccountOnMount
+          reason="transaction-flow-confirmation"
+          priority={10}
+          accountId={account.id}
+        />
+        {neurons.fullNeurons.length > 0 ? (
+          <WarnBox>
+            <Trans i18nKey="internetComputer.manageNeuronFlow.listNeuron.warnbox" />
+          </WarnBox>
+        ) : null}
+        <List
+          neurons={{
+            fullNeurons: neurons.fullNeurons.sort(
+              (a, b) => Number(b.cached_neuron_stake_e8s) - Number(a.cached_neuron_stake_e8s),
+            ),
+            lastUpdatedMSecs: neurons.lastUpdatedMSecs,
+          }}
+          modalName="MODAL_ICP_LIST_NEURONS"
+          unit={unit}
+          onClickManage={onClickManage}
+        />
+      </Container>
+    );
+  }
+
+  if (error) {
+    return (
+      <Container shouldSpace={signed}>
+        <TrackPage
+          category="Manage Neurons ICP Flow"
+          name="Step Confirmation Error"
+          flow="stake"
+          action="listNeurons"
+          currency={currencyId}
+        />
+        {signed ? (
+          <BroadcastErrorDisclaimer title={t("internetComputer.confirmation.broadcastError")} />
+        ) : null}
+        <ErrorDisplay error={error} withExportLogs />
+      </Container>
+    );
+  }
+
+  return null;
+}
+
+export function StepListNeuronFooter({
+  account,
+  onClose,
+  transitionTo,
+  neurons,
+  onChangeTransaction,
+  setLastManageAction,
+  error,
+}: StepProps) {
+  const currencyName = account.currency.name;
+  const onClickSync = useCallback(() => {
+    const bridge = getAccountBridge(account, undefined);
+    const initTx = bridge.createTransaction(account);
+    setLastManageAction("list_neurons");
+    onChangeTransaction(
+      bridge.updateTransaction(initTx, {
+        type: "list_neurons",
+      }),
+    );
+    transitionTo("device");
+  }, [account, onChangeTransaction, transitionTo, setLastManageAction]);
+
+  const onRetry = useCallback(() => {
+    transitionTo("manage");
+  }, [transitionTo]);
+
+  return (
+    <Box width="100%" horizontal alignItems="center" justifyContent="space-between">
+      <Box ff="Inter|SemiBold" fontSize={4} color="palette.text.shade60">
+        {`Last Synced: ${neurons.lastUpdatedMSecs ? new Date(neurons.lastUpdatedMSecs).toLocaleString() : "Never"}`}
```

Comment:
`Last Synced` and `Never` should be translated, if our application supports localised content.